recipes/libpq/all/conanfile.py

```diff
@@ -52,8 +54,11 @@ def export_sources(self):
 
     def config_options(self):
         if self.settings.os == "Windows":
-            self.options.rm_safe("fPIC")
-            self.options.rm_safe("disable_rpath")
+            del self.options.fPIC
+            del self.options.disable_rpath
+            del self.options.with_icu
+        if Version(self.version) < "16.0":
+            del self.options.with_icu
```

Comment:
```
======== Computing dependency graph ========
ERROR: libpq/15.5: Error in config_options() method, line 61
	del self.options.with_icu
	ConanException: option 'with_icu' doesn't exist
Possible options are ['shared', 'with_openssl']
```

On Windows this would be a double delete